provisioning/baremetal_pod.go

```diff
@@ -29,23 +29,26 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
+	configv1 "github.com/openshift/api/config/v1"
+
 	metal3iov1alpha1 "github.com/openshift/cluster-baremetal-operator/api/v1alpha1"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
 	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
 )
 
 const (
-	metal3AppName              = "metal3"
-	baremetalDeploymentName    = "metal3"
-	baremetalSharedVolume      = "metal3-shared"
-	metal3AuthRootDir          = "/auth"
-	ironicCredentialsVolume    = "metal3-ironic-basic-auth"
-	ironicrpcCredentialsVolume = "metal3-ironic-rpc-basic-auth"
-	inspectorCredentialsVolume = "metal3-inspector-basic-auth"
-	htpasswdEnvVar             = "HTTP_BASIC_HTPASSWD" // #nosec
-	mariadbPwdEnvVar           = "MARIADB_PASSWORD"    // #nosec
-	cboOwnedAnnotation         = "baremetal.openshift.io/owned"
-	cboLabelName               = "baremetal.openshift.io/cluster-baremetal-operator"
+	metal3AppName                    = "metal3"
+	baremetalDeploymentName          = "metal3"
+	baremetalSharedVolume            = "metal3-shared"
+	metal3AuthRootDir                = "/auth"
+	ironicCredentialsVolume          = "metal3-ironic-basic-auth"
+	ironicrpcCredentialsVolume       = "metal3-ironic-rpc-basic-auth"
+	inspectorCredentialsVolume       = "metal3-inspector-basic-auth"
+	htpasswdEnvVar                   = "HTTP_BASIC_HTPASSWD" // #nosec
+	mariadbPwdEnvVar                 = "MARIADB_PASSWORD"    // #nosec
+	cboOwnedAnnotation               = "baremetal.openshift.io/owned"
+	cboLabelName                     = "baremetal.openshift.io/cluster-baremetal-operator"
+	externalTrustBundleConfigMapName = "cbo-trusted-ca"
```

Comment:
What is the reason for duplication? Why can't we use the mao-trusted-ca configmap here? AFAIK, CBO and MAO need to live in the same namespace.

Reply:
Cross-repo dependency seems like a bad idea, I don't want to rely on MAO's config map. What if they remove it or change the name?